File: TUGAS DPBO/Latihan 1/B/C++/main.cpp
#include <bits\stdc++.h>
#include "club.cpp"

using namespace std;

int main(){
    int n=0;
    cin >> n;
    club arr[n];

    int i=0;
    string name, year, country, player, coach;
    for(i=0; i<n; i++){
        cout << "Input Club : ";
        cin >> name;

        cout << "Input Year : ";
        cin >> year;

        cout << "Input Country : ";
        cin >> country;

        cout << "Input player : ";
        cin >> player;

        cout << "Input Coach : ";
        cin >> coach;

        arr[i].setName(name);
        arr[i].setYear(year);
        arr[i].setCountry(country);
        arr[i].setPlayer(player);
        arr[i].setCoach(coach);

        arr[i] = club(name,year,country,player,coach);
    }

    for(i=0; i<n; i++){
        cout << endl << "Club : " << arr[i].getName() << endl;
        cout << "Year : " << arr[i].getYear() << endl;
        cout << "Country : " << arr[i].getCountry() << endl;
        cout << "Player : " << arr[i].getPlayer() << endl;
        cout << "Coach : " << arr[i].getCoach() << endl;
    }

    return 0;
}

File: TUGAS DPBO/Latihan 1/B/python/main.py
from club import club
n = int(input())

arr = [club() for i in range(n)]
i = 0

for i in range(n):
    # name, country, year, player, coach  = map(int, input().split())
    print("Input Name : ",end='')
    name = str(input())
    
    print("Input Country : ",end='')
    country = str(input())

    print("Input Year : ",end='')
    year = str(input())

    print("Input Player : ",end='')
    player = str(input())

    print("Input Coach : ",end='')
    coach = str(input())

    arr[i].setName(name)
    arr[i].setCountry(country)
    arr[i].setYear(year)
    arr[i].setPlayer(player)
    arr[i].setCoach(coach)

i = 0
for i in range(n):
    print()
    print("Club : ", str(arr[i].getName()))
    print("Country : ", str(arr[i].getCountry()))
    print("Year : ", str(arr[i].getYear()))
    print("Player : ", str(arr[i].getPlayer()))
    print("Coach : ", str(arr[i].getCoach()))